<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShankMapping</class>
 <widget class="QDialog" name="ShankMapping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>504</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Shank Map (Acq chan -&gt; probe site)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>15</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="nslbl">
         <property name="text">
          <string>NS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nsSB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nclbl">
         <property name="text">
          <string>NC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ncSB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nrlbl">
         <property name="text">
          <string>NR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nrSB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="chanLbl">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>71</red>
             <green>3</green>
             <blue>34</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>71</red>
             <green>3</green>
             <blue>34</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>106</red>
             <green>104</green>
             <blue>100</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Chans: 9999</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <row>
      <property name="text">
       <string>0</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Shank</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Col</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Row</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Used</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="statusLbl">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>3</green>
           <blue>34</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>3</green>
           <blue>34</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>18</number>
     </property>
     <property name="rightMargin">
      <number>18</number>
     </property>
     <item>
      <widget class="QPushButton" name="loadBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>nsSB</tabstop>
  <tabstop>ncSB</tabstop>
  <tabstop>nrSB</tabstop>
  <tabstop>defaultBut</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>loadBut</tabstop>
  <tabstop>saveBut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
